php/getBudgetTable.php

```diff
@@ -85,13 +86,13 @@
 			<tbody>
 				<?php
 					// add procedure rows
-					$columns = count($arm->visits);
+					$columns = count($arm['visits']);
 					foreach ($procedures as $proc_i => $procedure) {
-						$proc_name = $procedure->name;
-						$proc_cost = $procedure->cost;
-                        if ($procedure->routine_care_procedure_form) {
-                            $proc_name .= ' <span title="Standard of Care">[SoC]</span>';
-                        }
+						$proc_name = $procedure['name'];
+						$proc_cost = $procedure['cost'];
+                        //if ($procedure['routine_care_procedure_form']) {
```

Comment:
Same here

Reply:
This is a pending feature so it's commented out until we figure out if we're going to use it for this version of the budget tool